README.md

```diff
@@ -1,6 +1,10 @@
 # Multiplayer
 Multiplayer mod for RimWorld
 
+This is the new repository from Zetrith's Multiplayer, as he has left and can not be reached we have decided to work on it ourselves.
+If you would like to help work on it we have many things to do, contact someone on the discord, https://discord.gg/89cYfU9
```

Comment:
If you would like to contribute, take a look over on the GitHub project boards here: https://github.com/Parexy/Multiplayer/projects. If you want to implement your own feature into the mod, please contact any staff over on the discord, https://discord.gg/89cYfU9